public/mvnv/config-html-MVNV.hjson

```diff
@@ -17,7 +17,8 @@
     }
     uiConfig: {
         contactEmail: "test@test.com"
-        helpTextPath: "mvnv/help.md"
+        helpTextPath: "mvnv/help.md",
+        helpImgPath: "mvnv/quickStart.png",

```

Comment:
Can you not reference this image from the help.md? If we could do that, it would mean it would be possible to embed many images, instead of being limited to 1